lib/models/Restaurant.js

```diff
@@ -0,0 +1,49 @@
+const pool = require('../utils/pool');
+const Review = require('./Review');
+
+module.exports = class Restaurant {
+  id;
+  name;
+  cuisine;
+  cost;
+  image;
+  website;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.cuisine = row.cuisine;
+    this.cost = row.cost;
+    this.image = row.image;
+    this.website = row.website;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query('SELECT * FROM restaurants');
+    return rows.map((row) => new Restaurant(row));
+  }
+
+  static async getById(id) {
+    const { rows } = await pool.query(
+      `
+      SELECT * 
+      FROM restaurants 
+      WHERE 
+      id = $1
+      `,
+      [id]
+    );
```

Comment:
It would be a good practice to add a guard clause here to ensure you don't try to construct a new Restaurant object with bad data (such as an empty array). You can add a line here to abandon ship if that was going to happen.

```js
if (!rows[0]) return null; 
```